moving_nest/bounding_box.F90

```diff
@@ -64,7 +70,12 @@ subroutine fill_bbox_r4_2d(out_bbox, in_grid)
     out_bbox%je = ubound(in_grid, 2)
   end subroutine fill_bbox_r4_2d
 
-
+  !> ???
```

Comment:
@wramstrom can you please tell us what should replace the ??? in this documentation?